packages/patternfly-4/react-core/src/components/ContextSelector/ContextSelectorItem.tsx

```diff
@@ -1,40 +1,38 @@
-import React from 'react';
-import PropTypes from 'prop-types';
+import * as React from 'react';
 import styles from '@patternfly/patternfly/components/ContextSelector/context-selector.css';
```

Comment:
Change to `@patternfly/react-styles/css/components/ContextSelector/context-selector`